### DISCUSSION TITLE
What is the "encoding" dependency in the Next.js starter for?

### DISCUSSION THREAD

**Discussion by heloineto** [supabase/supabase#26492] [Questions] [answered]
The Next.js starter (https://github.com/vercel/next.js/blob/canary/examples/with-supabase/package.json) has a seemingly unused dependency "encoding".

Is it used? What is it for?

**Comment by GaryAustin1** [comment, marked as answer]
If you look at the blame for that line:
https://github.com/vercel/next.js/commit/2b497413087c56dedb3e04f77ea0deebb5d12d0b
![image](https://github.com/supabase/supabase/assets/54564956/3fcd49db-c2e9-4787-946f-ed8f75caac07)

**Reply by heloineto** [reply]
Thanks for saving me again!
